en-US/firefox/whatsnew_66.lang

```diff
@@ -81,12 +86,19 @@ Try Firefox Send
 Create a Firefox Account
 
 
-;Already have an account?
-Already have an account?
+## TAG: wnp66_log_in
+;Have an account?
+Have an account?
```

Comment:
I completely missed how the string was exposed. It should have been a single string, not two pieces glued together with a space. Something like `Have an account? <a href="$(url)s">Log in</a>`.

Can you check with the developers if it can be fixed?

Reply:
Let me ask ping her.

Reply:
Somehow the changes didn't make any difference in the extracted strings. I still get two separate strings. Could you confirm? 
https://github.com/mozilla/bedrock/pull/6941/files

Reply:
> Somehow the changes didn't make any difference in the extracted strings. I still get two separate strings. Could you confirm?
> https://github.com/mozilla/bedrock/pull/6941/files 

For me it extracts as 

```
Have an account? <a %s >Log in</a>
```

> it doesn't look quite like yours in the source.

That's OK, they use a variable to replace a lot more than the URL